packages/block-editor/src/components/url-input/index.js

```diff
@@ -212,6 +218,26 @@ class URLInput extends Component {
 		}
 	}
 
+	onFocus( event ) {
+		const { suggestions } = this.state;
+		const { disableSuggestions, onFocus } = this.props;
+
+		const inputValue = event.target.value;
```

Comment:
In what way would we expect this not to be the same as `this.props.value`?

Reply:
Great point. I was following the `onChange` implementation. I can replace `event.target.value` with `this.props.value` (or, rather `value` and add `value` to `const { disableSuggestions, onFocus, value } = this.props`).

I will add that in to the same PR that the documentation update is in, unless you'd rather they be separated out.

Reply:
Ultimately, I question whether we need the argument _at all_.

With `onChange`, the value represents some _intent to change_ the value. With `onFocus`, I don't think we're wanting to effect any change in the value?

Reply:
Completely agree. Thanks for walking me through this so clearly. Learning a lot from your comments! 💯